src/main/java/org/candlepin/subscriptions/tally/admin/InternalTallyResource.java

```diff
@@ -272,10 +272,19 @@ public DefaultResponse tallyConfiguredOrgs() {
 
   /** Trigger a tally for an org */
   @Override
-  public DefaultResponse tallyOrg(String orgId) {
+  public DefaultResponse tallyOrg(String orgId, Boolean xRhSwatchSynchronousRequest) {
     Object principal = ResourceUtils.getPrincipal();
     log.info("Tally for org {} triggered over API by {}", orgId, principal);
-    internalTallyDataController.tallyOrg(orgId);
+    if (Boolean.TRUE.equals(xRhSwatchSynchronousRequest)) {
+      if (!applicationProperties.isEnableSynchronousOperations()) {
+        throw new BadRequestException("Synchronous tally operations are not enabled.");
+      }
```

Comment:
[Optional] can we have log message here similar to https://github.com/RedHatInsights/rhsm-subscriptions/blob/main/src/main/java/org/candlepin/subscriptions/tally/admin/InternalTallyResource.java#L118 
saying `Synchronous tally requested for orgId` ?

Reply:
+1 added